nailgun/entities.py

```diff
@@ -3397,6 +3402,25 @@ def read(self, entity=None, attrs=None, ignore=None):
         ignore.add('discovery')
         return super(Subnet, self).read(entity, attrs, ignore)
 
+    def update_payload(self, fields=None):
+        """Wrap submitted data within an extra dict."""
+        return {
```

Comment:
As far as I see, it can be one line statement:
`return {u'subnet': super(Subnet, self).update_payload(fields)}`